### DISCUSSION TITLE
What is your favorite lunch?

### DISCUSSION THREAD

**Discussion by Kikuchi-Takamasa** [NAIST-SE/AttractingContributors#1072] [2024 Attraction Contributions]
Hi, everyone.

I've started a new repository to share and discuss our favorite lunch. If you have a lunch that you love and want to recommend, please join the project and add it to the list! Looking forward to seeing your contributions!

Repository: https://github.com/Kikuchi-Takamasa/AttractingContributors/blob/main/README.md

**Comment by gill876** [comment]
Mine is katsudon!

Please make a contribution to my project!
https://github.com/NAIST-SE/AttractingContributors/discussions/1089#discussion-6846489

**Comment by KazumaYamasaki** [comment]
I added `Akashiyaki`!
I would be happy if you could also send a pull request to my repository!
↓
https://github.com/KazumaYamasaki/AttractingContributors-Keyboard

**Comment by yuanhau-cheng** [comment]
I have submit a pull request. If you have time, please also help me, Thanks a lot! (Empty pull request also ok I think)
This is my link:
https://github.com/yuanhau-cheng/Japanese-Role-Playing-Game
